packages/hash/api/src/model/knowledge/orgMembership.model.ts

```diff
@@ -0,0 +1,170 @@
+import { GraphApi } from "../../graph";
+import {
+  OrgMembershipModel,
+  EntityModel,
+  EntityModelCreateParams,
+  OrgModel,
+  UserModel,
+} from "..";
+import { workspaceAccountId } from "../util";
+import { WORKSPACE_TYPES } from "../../graph/workspace-types";
+
+export type OrgMembershipModelCreateParams = Omit<
+  EntityModelCreateParams,
+  "properties" | "entityTypeModel" | "accountId"
+> & {
+  responsibility: string;
+  org: OrgModel;
+};
+
+/**
+ * @class {@link OrgMembershipModel}
+ */
+export default class extends EntityModel {
+  /**
+   * Create a workspace OrgMembership entity.
+   *
+   * @param params.responsibility - the role of the user at the organization
+   */
+  static async createOrgMembership(
+    graphApi: GraphApi,
+    params: OrgMembershipModelCreateParams,
+  ) {
+    const { responsibility, org } = params;
+
+    const properties: object = {
+      [WORKSPACE_TYPES.propertyType.responsibility.baseUri]: responsibility,
+    };
+
+    const entityTypeModel = WORKSPACE_TYPES.entityType.orgMembership;
+
+    const entity = await EntityModel.create(graphApi, {
+      accountId: workspaceAccountId,
```

Comment:
Oh I hadn't thought about how this means org memberships are also under the account. Kinda gross 🙃.

I guess we can revisit later

Reply:
Yeah, it also seemed a bit strange to me but sort of makes sense as I don't think users should be the "owners" of this entity. Maybe the org can be considered the "owner"? But yeah let's revisit later if it becomes important.